pgo/cmd/cluster.go

```diff
@@ -331,6 +332,17 @@ func createCluster(args []string, ns string, createClusterCmd *cobra.Command) {
 	if createClusterCmd.Flag("sync-replication").Changed {
 		r.SyncReplication = &SyncReplication
 	}
+	// only set BackrestS3VerifyTLS in the request if actually provided via the CLI
+	// if set, store provided value accordingly
+	if createClusterCmd.Flag("pgbackrest-s3-verify-tls").Changed {
+		if BackrestS3VerifyTLS {
+			r.BackrestS3VerifyTLS = msgs.UpdateBackrestS3VerifyTLSEnable
+		} else {
+			r.BackrestS3VerifyTLS = msgs.UpdateBackrestS3VerifyTLSDisable
+		}
+	} else {
+		r.BackrestS3VerifyTLS = msgs.UpdateBackrestS3VerifyTLSDoNothing
+	}
```

Comment:
Recommendation to simplify:

```suggestion
	r.BackrestS3VerifyTLS = msgs.UpdateBackrestS3VerifyTLSDoNothing
	
	if createClusterCmd.Flag("pgbackrest-s3-verify-tls").Changed {
		if BackrestS3VerifyTLS {
			r.BackrestS3VerifyTLS = msgs.UpdateBackrestS3VerifyTLSEnable
		} else {
			r.BackrestS3VerifyTLS = msgs.UpdateBackrestS3VerifyTLSDisable
		}
	}
```

Reply:
That works for me, updated. Thanks!